components/great-expectations/validate/CSV/component.py

```diff
@@ -42,9 +44,10 @@ def validate_csv_using_greatexpectations(
 
 
 if __name__ == '__main__':
-    calculate_regression_metrics_from_csv_op = create_component_from_func(
+    validate_csv_using_greatexpectations = create_component_from_func(
```

Comment:
Are you sure? The function returned from `create_component_from_func` is slightly different from the one passed in. It might be better to give them different names.

Reply:
Sure. The variable name isn't important here because it's not used, but we need to use different names for pipeline steps and functions for consistency. I've change the variable name to a more appropriate one.